CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.139.0, 30 March 2021
+
+- Bundler [Beta]: Detect and run Bundler V1 or V2 based on Gemfile.lock
+  - Requires `{ options: { bundler_2_available: true }}` to be passed for this release
```

Comment:
```suggestion
  - Requires `options: { bundler_2_available: true }` to be passed to Bundler classes for this release
```

Reply:
Seems ok 👍  could clarify it's needed for FileParser, UpdateChecker and FileUpdater? I think those are the ones it's used in